vpr/src/pack/cluster_router.cpp

```diff
@@ -582,7 +590,13 @@ void free_intra_lb_nets(std::vector<t_intra_lb_net>* intra_lb_nets) {
  ****************************************************************************/
 
 /* Recurse through route tree trace to populate pb pin to atom net lookup array */
-static void load_trace_to_pb_route(t_pb_routes& pb_route, const int total_pins, const AtomNetId net_id, const int prev_pin_id, const t_lb_trace* trace) {
+static void load_trace_to_pb_route(t_pb_routes& pb_route,
```

Comment:
Where is the "pb pin to atom net lookup array" stored?  Inside pb_route?  This should be documented in a bigger Doxygen comment for this routine.